server/room.py

```
@@ -203,6 +206,63 @@ def tick(conn: sqlite3.Connection, store: SessionStore) -> None:
 
+#: תקרה לאורך פרט ביומן. הודעת חריגה יכולה לגרור traceback שלם, ושורת
+#: יומן שאי אפשר לקרוא על המסך אינה מוסיפה למפעיל דבר.
+_DETAIL_MAX = 200
+
+
+def _one_line(value: object) -> str:
+    """ערך כפי שמותר לכתוב אותו לשורת יומן או לוג: **שורה אחת**.
+
+    **לא** מפני שהוא מזייף רשומה בטבלת היומן — הוא אינו יכול:
+    ‏`journal()` מעביר את `detail` כפרמטר קשור לעמודת TEXT, ושורה חדשה
+    בתוכו נשארת שדה אחד. בקרה שלילית שהייתה נשענת על "רשומה שנייה
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)